src/plugin_manager.py

```diff
@@ -311,7 +311,8 @@ def plugin_view(self):
 
         #Set the folder size
         new_width, new_height = plugin_widget.width / 4 - 10, plugin_widget.width / 4 - 10
-        self.plugin_player.create_image('folder_icon.png', 'button_folder.png',
+        self.plugin_player.create_image('../assets/folder_icon.png',
+                                        '../assets/button_folder.png',
```

Comment:
Make sure to propagate this name change to the `Button` below.

Reply:
I've changed this to work when ran in the PluginPlayer folder, if it is ran in src, no pictures will be loaded, I assume this is not an issue, if it is I will need to look further into how to standardize the path when working in different directories.